ldi-core/version-object-creator/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/VersionObjectCreator.java

```diff
@@ -47,8 +47,11 @@ private MemberInfo extractMemberInfo(Model linkedDataModel, Resource memberTypeR
 			Property dateObservedProperty) {
 		String dateObserved = Optional.ofNullable(dateObservedProperty)
 				.map(property -> linkedDataModel.listStatements(null, property, (Resource) null)
+						.mapWith(Statement::getObject)
```

Comment:
You could use "listObjectsOfProperty" instead of list the statements and then mapping them to objects.